frontend/src/components/admindashboard/admindashboard.js

```diff
@@ -16,13 +17,22 @@ const Admindashboard = () => {
 
   const fetchData = (pageNumber) => {
     axios
-      .get(REG_URL + '?page=' + pageNumber)
+      .get(CAT_URL + '?page=' + pageNumber)
```

Comment:
```suggestion
      .get(`${CAT_URL}?page=${pageNumber}`)
```

Reply:
Pls apply this to all axios routes.
Reference: [ Template literals (Template strings)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)